src/js/Edit/L.PM.Edit.CircleMarker.js

```diff
@@ -0,0 +1,131 @@
+import Edit from './L.PM.Edit';
+
+Edit.CircleMarker = Edit.extend({
+  initialize(layer) {
+    this._layer = layer;
+    this._enabled = false;
+  },
+  toggleEdit(options) {
+    if (!this.enabled()) {
+      this.enable(options);
+    } else {
+      this.disable();
+    }
+  },
+  enabled() {
+    return this._enabled;
+  },
+  enable(options) {
+    L.Util.setOptions(this, options);
+
+    this._map = this._layer._map;
+
+    // cancel when map isn't available, this happens when the polygon is removed before this fires
+    if (!this._map) {
+      return;
+    }
+
+    if (!this.enabled()) {
+      // if it was already enabled, disable first
+      // we don't block enabling again because new options might be passed
+      this.disable();
+    }
+
+    // change state
+    this._enabled = true;
+
+    // enable removal for the marker
+    if (!this.options.preventMarkerRemoval) {
+      this._layer.on('contextmenu', this._removeMarker, this);
+    }
+
+    // // init markers
+    this._createHintMarker();
```

Comment:
It seems overkill to have hint markers for circleMarkers to drag them. Please use 

```js
if (this.options.draggable) {
  this.enableLayerDrag();
}
```

to make circleMarkers draggable